### ISSUE TITLE
Almost unusable

### ISSUE THREAD

**Issue by JonSilver** [Alia5/lovelace-expander-card#34]
Almost unusable due to constant re-renders. I try to edit content and unless I'm extremely quick, everything redraws and I'm kicked out of what I was editing.

**Comment by marwonn** [comment]
Same here.

**Comment by Josephtsessions** [comment]
Same here sadly

**Comment by Ethan-G** [comment]
Yes this is driving me insane, which is a shame because it's otherwise pretty great. I've found if you make an edit in the code editor, it rerenders once then stops, allowing for more stress-free editing - but then you're stuck in the YAML.

**Comment by johnwalk61** [comment]
Same issue here. Redraws 5 seconds into the edit, and the code reverts to an old version.

**Comment by clucluclu** [comment]
Aah bummer, having same issue. A shame, since this card really seemed to be the perfect solution!

**Comment by Californian** [comment]
My workaround is to edit in yaml.

**Comment by yjamal01** [comment]
Same issue here, I really love this card so I have to edit in YAML to be able to keep using it

**Comment by FlexxFR** [comment]
same here, it's really hard to use like this.

**Comment by yjamal01** [comment]
I sponsored the creator, hoping it motivates him to update this as my dashboard relies on it heavily

**Comment by hvorragend** [comment]
I am using the following fork for several weeks now. 
And as far as I know I don't have any problems.
https://github.com/MelleD/lovelace-expander-card

**Comment by yjamal01** [comment]
> I am using the following fork for several weeks now. And as far as I know I don't have any problems. https://github.com/MelleD/lovelace-expander-card

I tried searching online but am having some trouble, could you explain for a layman how to use this forked repository? If I follow the installation instructions it says I already have the custom repository as the instructions on the one you posted are the same as this one. 

TYVM

**Comment by johnwalk61** [comment]
The guys repository instructions say to add the original git. Use the url for the replacement one.
----------------------------------------

*From: *yjamal01 ***@***.***>
*To: *Alia5/lovelace-expander-card ***@***.***>
*CC: *John Walker ***@***.***>; Comment ***@***.***>
*Date: *Nov 17, 2023 5:18:44 p.m.
*Subject: *Re: [Alia5/lovelace-expander-card] Almost unusable (Issue #34)

**Comment by yjamal01** [comment]
> I am using the following fork for several weeks now. And as far as I know I don't have any problems. https://github.com/MelleD/lovelace-expander-card

Same issue, are you only editing in YAML? Editing YAML works fine but if you try and use the GUI it is unusable

**Comment by m00nyONE** [comment]
got the same issue with both repositories here too. So i sadly need to uninstall this :-(

**Comment by Zerreth** [comment]
Whilst editing, it deleted everything in my horizontal-stack and replaced it with copies of itself. Awesome!

**Comment by yjamal01** [comment]
just hit cancel when it does that

> Whilst editing, it deleted everything in my horizontal-stack and replaced it with copies of itself. Awesome!

**Comment by milandzuris** [comment]
@Alia5

**Comment by Anabcent** [comment]
I am using the following fork https://github.com/spencermamer/lovelace-yet-another-expander-card which seems to work
